targets/TARGET_NUVOTON/TARGET_M480/crypto/crypto-misc.cpp

```diff
@@ -121,42 +140,42 @@ void crypto_zeroize32(uint32_t *v, size_t n)
 
 bool crypto_aes_acquire(void)
 {
-    return crypto_submodule_acquire(&crypto_aes_avail);
+    return crypto_aes_mutex->lock() == osOK;
```

Comment:
No need for this check - indeed it's deprecated.

         @note This function treats RTOS errors as fatal system errors, so it can only return osOK.
         Use of the return value is deprecated, as the return is expected to become void in the future.

Reply:
@kjbracey-arm OK. I remove the return code check and introduce `xxx_try_acquire` to make code more clear.